alcs-api/src/planning-review/planning-review.module.ts

```diff
@@ -0,0 +1,22 @@
+import { forwardRef, Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { BoardModule } from '../board/board.module';
+import { CardModule } from '../card/card.module';
+import { CodeModule } from '../code/code.module';
+import { PlanningReviewProfile } from '../common/automapper/planning-meeting.automapper.profile';
+import { PlanningReviewController } from './planning-review.controller';
+import { PlanningReview } from './planning-review.entity';
+import { PlanningReviewService } from './planning-review.service';
+
+@Module({
+  imports: [
+    TypeOrmModule.forFeature([PlanningReview]),
+    forwardRef(() => BoardModule),
```

Comment:
If I'm not mistaken, forwardRef is needed only on one side of modules that cause circular dependencies